### DISCUSSION TITLE
Is it possible to use a remote Borg-Server with the built-in Backup-Solution?

### DISCUSSION THREAD

**Discussion by Schema-F** [nextcloud/all-in-one#1426] [Questions] [answered]
The Readme says someone should Backup on the host and rsyncing. But this means i need the used space three times (productive, backup-host, backup-remote). Borg-Backup can backup direct to a remote server, if there´s borg installed (user@host:/path_to_repo). Is it possible to use this with the built-in Backup-Solution?

**Comment by szaimen** [comment, marked as answer]
No, that is not possible. See https://github.com/nextcloud/all-in-one/discussions/1068#discussioncomment-3486952
